gtk/Gir.toml

```diff
@@ -2706,14 +2706,12 @@ manual_traits = ["WidgetExtManual"]
         name = "data"
         transformation = "borrow"
     [[object.signal]]
-    name = "drag-drop"
-    inhibit = true
+    name = "drag-drop" # this bool return should not be an Inhibit
     [[object.signal]]
-    name = "drag-failed"
+    name = "drag-failed" # this one should
     inhibit = true
     [[object.signal]]
-    name = "drag-motion"
-    inhibit = true
+    name = "drag-motion" # this one should not
```

Comment:
You can just drop this

Reply:
There is an existing comment above:
```rust
# return Inhibit instead of bool
```
which is implied to cover all the subsequent signals. I put those comments in to make it clear that those are individual, deliberate exceptions.

(I would have reordered them, which would have allowed them to be clearly delineated with a single comment, but that would have made a much messier diff, and I would have worried more about breaking something.)

Reply:
oh, you were saying that those two entire `[[object.signal]]` sections can be dropped. (Sorry, I am sleep deprived and slow on the uptake today.)

Reply:
No worries, thanks!